Added Deploy to Azure button to Azure Arc-enabled servers scenarios (#1248)
microsoft/azure_arc: docs/azure_arc_jumpstart/azure_arc_servers/day2/arc_monitoring/_index.md

- [Install or update Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli?view=azure-cli-latest). Azure CLI should be running version 2.14 or later. Use ```az --version``` to check your current installed version.

## Deployment Options and Automation Flow

This Jumpstart scenario provides multiple paths for deploying and configuring resources. Deployment options include:

- Azure portal
- ARM template via Azure CLI

> For you to get familiar with the automation and deployment flow, below is an explanation.

Reviewer: ```suggestion
The steps below will help you get familiar with the automation and deployment flow.
```

> - User is providing the ARM template parameters values, either via the portal or editing the parameters file (1st-time edit). These parameters values are being used throughout the deployment.

Reviewer: ```suggestion
- User provides the ARM template parameter values, either via the portal or editing the parameters file. These parameter values are being used throughout the deployment.
```
